docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,808 @@
+- Metric gateway configured with a broken exporter to simulate backpressure from metric gateway to metric agent
+
+
+### Findings
+- Test results with a single agent instance and single receiver configuration: A single target scrape reached max ~120K metric data point.
+- Single agent instance with single receiver reached ~280K metric points per scrape (~14K metric point/sec) and stayed stable. For anything above, the memory_limiter processor refuses the metric data.
+- Multi-Node test with single receiver performed on 15 Nodes cluster 1430 Pods (this was the max Pod count can be deployed on this cluster): Agent was able to scrape in total ~2.800K metrics data per scrape loop, and data was pushed successfully to the agent without any data dropping by agent side.
+  - No memory issues identified and no additional memory impact found.
+  - Kubernetes API server request duration for `GET` operations reached ~750ms and for `LIST` operations ~900ms. There was only one short peak of 21s detected for the `GET` operation.
+- `sample_limit` configuration tested for multiple receivers. Configured limitation works but there are no metrics identified to see limit violations when they occur, only logs are present in this case.
+- Multi-receiver test reached same stable limit of ~280K metrics per scrape in total, to test OOM resilience of setup, 45% more load added on peak stable load. Metric agent stay stable over 12 hour running test, no OOM observed, load over ~280K refused successfully.
+- In all test cases backpressure from broken metric gateway handled by the metric agent successfully, no serious impact on metric agent observed, metric agent exporter successfully queued metric data and drop after unsuccessfully retry. 
+
+## Conclusion
+
+To achieve a stable and resilient metric agent setup, the following configuration parameters must be added or adjusted.
+
+To avoid metric agent to overrun with a huge amount of data floods from single scrape target and run in OOM, configuration parameter `sample_limit: 50000` were added to each Prometheus receiver scrape configuration.
+The metric agent can handle higher values, but analyzing current Kyma production deployments shows values around 50000 in average.
+
+For more memory resilience, the processor memory_limiter was configured with a hard limit of 75%,  and hard and soft memory limit check intervals were configured to 0.5 second to allow the processor to react faster when limits are exceed.
+
+Batch processor was configured with a batch size of 1024 to avoid hitting the gRPC client payload size limit while metrics are exported to the gateway instances.
+Default batch size of 8192 was over the default gRPC client payload limit of 4MByte.
+
+By the multi-receiver test some OOM observed, after further analysis result following finding:
+1. Under high load, memory limiter trigger GC to free memory and go back for normal operation
```

Comment:
I doubt that the order of findings matters, so a bullet list is better

```suggestion
- Under high load, memory limiter triggers GC to free memory and go back for normal operation.
```

Who is "GC" ?